fea-rs/src/compile/features.rs

```diff
@@ -46,6 +69,15 @@ pub(crate) struct SizeFeature {
     pub names: Vec<NameSpec>,
 }
 
+#[derive(Clone, Debug, Default)]
+pub(crate) struct CvParams {
+    pub feat_ui_label_name: Vec<NameSpec>,
+    pub feat_ui_tooltip_text_name: Vec<NameSpec>,
+    pub samle_text_name: Vec<NameSpec>,
```

Comment:
`sample_text_name`. One day I’ll get around to fixing the `varations` field in the CFF code :)